### PR TITLE
Add perfect-numbers exercise

### PR DESCRIPTION
**Pull request by haus**
This adds the perfect-numbers exercise, with tests borrowed from the
ruby implementation.

### CODE REVIEW

**Comment by yurrriq** [comment]
Can you please rebase and decide whether to make that cosmetic edit? I'll merge after that.

File: exercises/perfect-numbers/src/example.clj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+(ns perfect-numbers)
+
+(defn- get-divisors
+  "Utility function to get the divisors of a number"
+  [number]
+  (for
+   [n (range 1 (inc (quot number 2)))
```
Comment (haus) [inline exercises/perfect-numbers/src/example.clj]: That seems fine to me. I'll make that change and rebase.

**Comment by haus** [comment]
@yurrriq rebased and made the requested change

File: config.json [hunk 2]
```diff
@@ -88,6 +88,11 @@
     },
     {
       "difficulty": 1,
+      "slug": "perfect-numbers",
+      "topics": []
+    },
+    {
+      "difficulty": 1,
```
Comment (yurrriq) [inline config.json]: Thanks for this, too.

**Comment by yurrriq** [comment]
Thanks!

**Pull request merged**